### DISCUSSION TITLE
changeset-bot not available for GitHub Enterprise?!

### DISCUSSION THREAD

**Discussion by LorenzKahl** [changesets/changesets#1213] [General]
Hi, 

we are running on GitHub Enterprise (on premise). 

As far as I can see the changeset-bot is not available for these kinds of accounts.
Is there any existing action or workflow to create something similar like the bot is doing?

**Comment by Andarist** [comment]
The code of that bot is [here](https://github.com/changesets/bot) so it should be possible to reuse the logic there using some other triggers.

**Reply by LorenzKahl** [reply]
Thank you. I will check that out!
